okareo_tests/test_integration_model_under_test.py

```diff
@@ -721,3 +723,109 @@ def test_run_test_predefined_checks(
     )
     assert run_resp.name == f"openai-chat-run-predefined-{rnd}"
     assert_metrics(run_resp, checks)
+
+
```

Comment:
There is enough here that it probably deserves it's own test class, rather than just adding to the super overloaded one.